src/Form/Type/PlanningSearchType.php

```diff
@@ -114,6 +114,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     'data-actions-box' => 'true',
                 ],
             ])
+            ->add('withoutAvailabilities', CheckboxType::class, [
```

Comment:
Starting the parameter value with "display", "only", or "hide" would be easier to understand. What do you think about `displayWithoutAvailabilities` ?